apptests/appscenarios/reloader_test.go

```diff
@@ -18,21 +18,20 @@ import (
 )
 
 var (
-	r                 *reloader
-	hr                *fluxhelmv2beta2.HelmRelease
-	deploymentList    *appsv1.DeploymentList
-	reloaderContainer corev1.Container
-	err               error
+	r                      *reloader
+	reloaderHr             *fluxhelmv2beta2.HelmRelease
+	reloaderDeploymentList *appsv1.DeploymentList
+	reloaderContainer      corev1.Container
 )
```

Comment:
Do these need to be global? I put them inside the `var _ = Describe("Reloader Install Test",` to prevent the simple names from conflicting in the broader scope.